RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -35,7 +35,7 @@
 trackingPhase2PU140.toModify(highPtTripletStepSeedLayers,
 # combination with gap removed as only source of fakes in current geometry (kept for doc) 
     layerList = ['BPix1+BPix2+BPix3', 'BPix2+BPix3+BPix4',
-#                 'BPix1+BPix3+BPix4', 'BPix1+BPix2+BPix4',
+                 'BPix1+BPix3+BPix4', 'BPix1+BPix2+BPix4',
```

Comment:
We tried to include just one but this was not enough to recover all the efficiency.